src/GUI/GUIFont.cpp

```diff
@@ -453,6 +453,9 @@ void GUIFont::DrawText(GUIWindow *pWindow, Pointi position, Color uFontColor, co
         return;
     }
 
+    if (uFontShadowColor == Color())
```

Comment:
There may be some instance where we don't want to shadow to show. In which case Color() would be valid.

Reply:
I'll leave a TODO then, this is a question of replacing `Color()` with `colorTable.Black` at all call sites.

Reply:
Btw, right now you can use `Color(255, 255, 255, 0)` if you don't want the shadow.